builder/dockerfile/parser/testfiles/continueIndent/Dockerfile

```diff
@@ -27,8 +27,5 @@ bye\
 frog
 
 RUN echo hello \
-# this is a comment
-
-# this is a comment with a blank line surrounding it
-
-this is some more useful stuff
+# this is a comment that breaks escape continuation
+RUN echo this is some more useful stuff
```

Comment:
This is the offending test change -- a comment by itself _should_ still be valid in the middle of a continuation (very, very common, useful pattern, supported by Docker for ages now).